### PR TITLE
nicholascassar/class-1/insert-licence

### PR DESCRIPTION
**Pull request by nscassar**
Added a LICENCE file.

### CODE REVIEW

**Comment by Right2Drive** [comment]
@nscassar There is nobody assigned to this PR

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
+ *
+ * This file is part of AgriTrader.
+ *
+ * AgriTrader can not be copied and/or distributed without the express
+ * permission of James Thorogood.
+ *******************************************************/
```
Comment (Right2Drive) [inline LICENCE]: # Requested change

Please leave a newline at eof

Comment (nscassar) [reply LICENCE]: Added a newline, as well as a comment about how all files in AgriTrader's directories and subdirectories also belong to AgriTrader.

**Comment by Right2Drive** [comment]
@nscassar It's all good, you don't have to put anything in the description if you don't want! And if you're just responding to comments you don't have to leave a review if you would prefer not to. Github has kind of a confusing interface regarding this, and it can get very cluttered if people initialize comments on a file w/o the review process, but just commenting on previous review comments should be fine. Dw it's on Github for being confusing :P 
Sorry for taking so long

**Pull request merged**